crates/torii/core/Cargo.toml

```diff
@@ -14,6 +14,7 @@ async-trait.workspace = true
 chrono.workspace = true
 dojo-types = { path = "../../dojo-types" }
 dojo-world = { path = "../../dojo-world" }
+torii-graphql = { path = "../graphql" }
```

Comment:
I think this creates a cyclic dependency? I would prefer not to include graphql logic in core, even if it means some level of duplication

Reply:
Ohh make sense now